backbone/src/main/res/values/colors.xml

```diff
@@ -9,11 +9,15 @@
     <color name="rsb_black_40">#66000000</color>
     <color name="rsb_white_60">#CCFFFFFF</color>
     <color name="rsb_warm_gray">#FF757575</color>
+    <color name="rsb_greyish_brown">#474747</color>
```

Comment:
Thanks for adding these, let's coordinate on Slack to ensure we all use the same values (maybe we can add a ticket to create a "Styles/Themeoverlay" for all these to reuse. Not 1st priority, but something we'd love to do sooner rather than later, like you suggested earlier this week.

Reply:
Yes, it'd be better if we have a branch to include these changes, but for now if this PR is good to go, then we can merge just to save time.